inc/namespace.php

```diff
@@ -81,12 +81,16 @@ function bootstrap() {
 		define( 'WP_CACHE', true );
 	}
 
-	add_filter( 'wp_mail_from', function ( string $email ) use ( $config ) : string {
+	add_filter( 'wp_mail_from', function () use ( $config ) : string {
+		$default_email = sprintf(
+			'noreply@%s',
+			wp_parse_url( network_home_url( '/' ), PHP_URL_HOST )
```

Comment:
This should probably exclude `www` by the way, not sure if it does by default?

Reply:
Ah crap. Will do a follow up.